socorro/lib/threaded_task_manager.py

```diff
@@ -156,67 +139,73 @@ def wait_for_empty_queue(self, wait_log_interval=0, wait_reason=""):
             seconds += 1
             time.sleep(1.0)
 
-    def _kill_worker_threads(self):
-        """This function coerces the consumer/worker threads to kill
```

Comment:
I took out all the suicide, abortion, and homocide language and replaced it with clearer language.